.github/workflows/buildv2.yml

```diff
@@ -0,0 +1,30 @@
+name: Building v2
+
```

Comment:
Do we need a new workflow file for each new version?

I feel like most of these can be done in the same just with different step names.

Reply:
Yeah, that sounds a bit cleaner.